route/service.go

```diff
@@ -0,0 +1,55 @@
+package route
+
+import (
+	"fmt"
+	"github.com/opentracing/opentracing-go/log"
+	"github.com/thunderdb/ThunderDB/consistent"
+	"github.com/thunderdb/ThunderDB/proto"
+)
+
+type DhtService struct {
+	thisNode proto.Node
+	hashRing *consistent.Consistent
+}
+
+func NewDhtService() *DhtService {
+	return &DhtService{
+		hashRing: consistent.New(),
+	}
+}
+
+type GetNeighborsReq struct {
+	NodeId  proto.NodeId
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
struct field NodeId should be NodeID